dnf-behave-tests/features/countme.feature

```diff
@@ -44,6 +44,42 @@ Feature: Better user counting
             | header     | value  |
             | User-Agent | libdnf |
 
+    @bz1676891
+    @destructive
+    @fixture.httpd
+    Scenario: User-Agent header is sent (RHEL)
+        Given I am running a system identified as the "Red Hat Enterprise Linux 8.2 generic"
```

Comment:
Although this PR is already closed, let me just clarify:
"generic" is the default value of `VERSION_ID` in `/etc/os-release` on RHEL, so you wouldn't need to specify it here (but it wouldn't hurt of course).